### DISCUSSION TITLE
Set default icon in Category/AccessoryInformation?

### DISCUSSION THREAD

**Discussion by bramboe** [HomeSpan/HomeSpan#362] [Q&A] [answered]
Hi!

I went through the tutorials, API documentation and other explainers but couldn't find if it's possible to set the default icon that is available per category device?

Have a great weekend!

**Comment by HomeSpan** [comment, marked as answer]
There is no way to directly control the icons since they are automatically selected by the Home App.  However, based on a lot of experimentation I have determined how the Device Category and the types of Services in the first functional Accessory (for a bridge device) influence the icon the Home App chooses.  See the comments section of tutorials 3 and 11 for a discussion of what I found based on iOS 15.  Note you may need to unpair your device, type 'H' into the CLI to generate a new device ID, and then re-pair your device to see changes to the icon.

Unfortunately, Apple keeps changing the way the Home App renders various Services and Characteristics with each new update.  My guess is iOS 16 will bring some fairly significant changes, which will likely require me to refresh some of the HomeSpan examples (for the third time!)
